app/services/container_topology_service.rb

```diff
@@ -80,10 +80,14 @@ def entity_status(entity, kind)
     when 'VM', 'Host' then entity.power_state.capitalize
     when 'Node'
       condition = entity.container_conditions.first
-      if condition.name == 'Ready' && condition.status == 'True'
-        'Ready'
+      if condition.nil?
+        'Unknown'
       else
-        'NotReady'
+        if condition.name == 'Ready' && condition.status == 'True'
+          'Ready'
+        else
+          'NotReady'
+        end
```

Comment:
Minor, it can be simplified with:

    if conditon.try(:name) == 'Ready' && condition.try(:status) == 'True'
      'Ready'
    else
      'NotReady'
    end